frontend/src/views/MyApplications.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+  <div class="my-application-forms">
+    <h1>내 지원서</h1>
+
+    <ApplicationFormItem
+      class="application-forms"
+      v-for="recruitment in matchRecruitments"
+      :key="recruitment.id"
+      :recruitment="recruitment"
+      :submitted="isSubmitted(recruitment.id)"
+    />
+
+    <footer>
+      <a class="logo" href="#"></a>
+    </footer>
+  </div>
+</template>
+
+<script>
+import * as RecruitmentApi from "../api/recruitments"
+import * as ApplicationApi from "../api/application-forms"
+import ApplicationFormItem from "@/components/ApplicationFormItem"
+
+export default {
+  components: {
+    ApplicationFormItem,
+  },
+  data: () => ({
+    applicationForms: [],
+    recruitments: [],
+  }),
+  async created() {
+    const token = this.$store.getters["token"]
+
+    try {
+      // todo: 백엔드 api에 따라 가능하면 한 번에 받아오도록 수정
+      Promise.all([
+        ApplicationApi.fetchMyApplicationForms(token),
+        RecruitmentApi.fetchMyRecruitments(token),
+      ]).then(values => {
+        const { data: applicationFormsData } = values[0]
+        const { data: recruitmentData } = values[1]
+        this.applicationForms = applicationFormsData
+        this.recruitments = recruitmentData
```

Comment:
```suggestion
        const { data: applicationFormsData } = values[0]
        const { data: recruitmentData } = values[1]
        this.recruitments = getMatchRecruitments(applicationFormsData, recruitmentData)
```
제 생각엔 백엔드 api 구현에 따르는 프런트 코드의 변경을 최소화하려면 상태로 갖는 `data`의 필드는 `recuritments` 하나면 될 것 같아요.
 `matchRecruitment` **메서드**를 활용하면 하나의 상태만 관리할 수 있겠네요.

후에 백엔드 api가 어떤식으로 구현되더라도 코드에서 정확히 이 부분만 바뀌면 될 것 같아서 변경을 최소화 할 수 있다고 생각하는데 어떻게 생각하시나요?

Reply:
상태를 갖는 `data`의 필드를 줄이면 변경을 최소화 할 수 있다는 부분은 동의합니다. `this.recruitments`와 `matchRecruitments`는 두 개가 있을 필요가 없겠네요.
다만 의미적으로 `requirements`가 내 지원서 목록 `item`의 `props`로 가는 부분이 애매해서 `appliedRequirements`로 수정했습니다 :)